source/operations/monitoring/metrics-and-alerts.rst

```diff
@@ -0,0 +1,396 @@
+
+Object and Bucket Metrics
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. metric:: minio_bucket_objects_size_distribution
+
+   Distribution of object sizes in the bucket, includes label for the bucket 
+   name.
+
+.. metric:: minio_bucket_usage_object_total
+
+   Total number of objects
+
+.. metric:: minio_bucket_usage_total_bytes
+
+   Total bucket size in bytes
+
+Replication Metrics
+~~~~~~~~~~~~~~~~~~~
+
+These metrics are only populated for MinIO clusters with 
+:ref:`minio-bucket-replication-serverside` enabled.
+
+.. metric:: minio_bucket_replication_failed_bytes
+
+   Total number of bytes failed at least once to replicate.
+
+.. metric:: minio_bucket_replication_pending_bytes
+
+   Total bytes pending to replicate.
+
+.. metric:: minio_bucket_replication_received_bytes
+
+   Total number of bytes replicated to this bucket from another source bucket.
+
+.. metric:: minio_bucket_replication_sent_bytes
+
+   Total number of bytes replicated to the target bucket.
+
+.. metric:: minio_bucket_replication_pending_count
+
+   Total number of replication operations pending for this bucket.
+
+.. metric:: minio_bucket_replication_failed_count
+
+   Total number of replication operations failed for this bucket.
+
+Cache Metrics
+~~~~~~~~~~~~~
+
+.. metric:: minio_cache_hits_total
+
+   Total number of disk cache hits
+
+.. metric:: minio_cache_missed_total
+
+   Total number of disk cache misses
+
+.. metric:: minio_cache_sent_bytes
+
+   Total number of bytes served from cache
+
+.. metric:: minio_cache_total_bytes
+
+   Total size of cache disk in bytes
+
+.. metric:: minio_cache_usage_info
+
+   Total percentage cache usage, value of 1 indicates high and 0 low, label
+   level is set as well
+
+.. metric:: minio_cache_used_bytes
+
+   Current cache usage in bytes
+
+Capacity Metrics
+~~~~~~~~~~~~~~~~
+
+.. metric:: minio_cluster_capacity_raw_free_bytes
+
+   Total free capacity online in the cluster.
+
+.. metric:: minio_cluster_capacity_raw_total_bytes
+
+   Total capacity online in the cluster.
+
+.. metric:: minio_cluster_capacity_usable_free_bytes
+
+   Total free usable capacity online in the cluster.
+
+.. metric:: minio_cluster_capacity_usable_total_bytes
+
+   Total usable capacity online in the cluster.
+
+.. metric:: minio_node_disk_free_bytes
+
+   Total storage available on a disk. The ``server`` and ``disk`` label identifies the host machine and disk respectively.
+
+.. metric:: minio_node_disk_total_bytes
+
+   Total storage on a disk. The ``server`` and ``disk`` label identifies the host machine and disk respectively.
+
+.. metric:: minio_node_disk_used_bytes
+
+   Total storage used on a disk. The ``server`` and ``disk`` label identifies the host machine and disk respectively.
+
+Lifecycle Management Metrics
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. metric:: minio_cluster_ilm_transitioned_bytes
+
+   Total amount of data transitioned using :ref:`tiering/transition lifecycle management rules <minio-lifecycle-management-tiering>`
```

Comment:
Should this be `Total number of bytes of data` ... to match up with how we talk about other metrics above?

Reply:
Total number of bytes is probably fine...here `of data` would definitely be redundant.